package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Lightstep",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=16"
+    "node": "16"
```

Comment:
The current dependencies use an outdated version of canvas that requires Node 16 for the tests to pass. Attempting to run the tests with Node 18 results in this failure:

![Screenshot 2023-06-12 at 6 06 44 AM](https://github.com/lightstep/lightstep-observability-datasource/assets/8461733/07995c30-c730-4839-b462-b4551ff786b7)